incubator/binding-grpc-kafka/src/main/java/io/aklivity/zilla/runtime/binding/grpc/kafka/internal/config/GrpcKafkaOptionsConfigAdapter.java

```diff
@@ -28,25 +28,37 @@
 
 public final class GrpcKafkaOptionsConfigAdapter implements OptionsConfigAdapterSpi, JsonbAdapter<OptionsConfig, JsonObject>
 {
-    private static final String LAST_MESSAGE_ID_FIELD_NAME = "last-message-id-field";
-    private static final String LAST_MESSAGE_ID_METADATA_NAME = "last-message-id-metadata";
+    private static final String FIELD_NAME = "field";
+    private static final String METADATA_NAME = "metadata";
```

Comment:
To be consistent with the other constants, these should be renamed to `RELIABILITY_FIELD_NAME` and `RELIABILITY_METADATA_NAME`, and placed after `RELIABILITY_NAME`.

Also, `IDEMPOTENCY_METADATA_NAME` should be added and placed after `IDEMPOTENCY_NAME`.